processor/transaction/package_tests/TestProcessTransactionNullValues.approved.json

```diff
@@ -229,6 +229,7 @@
                     "us": 3781
                 },
                 "name": "SELECT FROM product_types",
+                "only_library_frames": false,
```

Comment:
I would expect the `only_library_frames` information to be part of the `stacktrace` which however would be a breaking change in the ES model and would require to re-introduce something like `frames`, e.g.
```
"stacktrace": {
  "only_library_frames": false,
  "frames": [ { .. }, ..  ]
}
```

@roncohen , @jalvz what do you say?

Reply:
I agree, but I'm fine either way.

Reply:
if we're going to add more stacktrace-specific metadata later it's better to bite the bullet now and make this change, which I agree with - @simitt @jalvz ?

Reply:
pushed up 925753f63148cff6f8ca9f8952df731b64ac9ad7 per @simitt's spec in https://github.com/elastic/apm-server/pull/392#discussion_r156291157, fine to remove that commit depending on @roncohen's decision